src/components/cta/cta.tsx

```diff
@@ -24,7 +24,44 @@ export class CtaComponent {
   // optional message prop
   @Prop() public message?: string;
```

Comment:
Should be renamed to text or displayText

Reply:
Sure , I will rename it to displaytext